config/install/encrypt.profile.tfa_encryption.yml

```diff
@@ -0,0 +1,12 @@
+langcode: en
```

Comment:
why did you split tfa-related configurations into two directories, `install` and `optional`?